wasm/text/parser.go

```diff
@@ -590,15 +611,16 @@ func (p *moduleParser) parseFuncBodyField(tok tokenType, tokenBytes []byte, line
 // and/or Func.typeInlined and sets the next parser to parse.
 func (p *moduleParser) parseFuncEnd(tok tokenType, tokenBytes []byte, _, _ uint32) error {
 	if tok == tokenRParen {
-		tu, paramNames := p.typeParser.getTypeUse()
+		tu, _, paramNames := p.typeParser.getTypeUse()
```

Comment:
this `_` will be used when we implement locals to range check.. if that doesn't end up true, I'll change the sig and delete it :D